packages/suite/src/views/wallet/nfts/NftsTable/NftsRow.tsx

```diff
@@ -0,0 +1,296 @@
+                            <Button
+                                icon="show"
+                                onClick={() => {
+                                    dispatch(
+                                        tokenDefinitionsActions.setTokenStatus({
+                                            symbol: network.symbol,
+                                            contractAddress: nft.contract || '',
+                                            status: TokenManagementAction.SHOW,
+                                            type: DefinitionType.NFT,
+                                        }),
+                                    );
+                                }}
+                                variant="tertiary"
+                                size="small"
+                            >
+                                <Translation id="TR_UNHIDE" />
+                            </Button>
+                        )}
+                    </Row>
+                </Table.Cell>
+            </Table.Row>
+            {nft.type === 'ERC721' &&
+                nft.ids?.map((id, index) => (
+                    <Table.Row
+                        key={id + index}
+                        isCollapsed={!isCollectionOpen}
+                        isHighlightedOnHover={false}
+                    >
+                        <Table.Cell colSpan={2}>
+                            <Text typographyStyle="hint">
+                                <HiddenPlaceholder>
+                                    <Row gap={spacings.xs}>
+                                        {pictureIcon}
+                                        <Text textWrap="nowrap">{NftName}</Text>
+                                        #<RedactNumericalValue value={id.toString()} />
+                                        <Badge size="small">
+                                            <RedactNumericalValue value="1x" />
+                                        </Badge>
+                                        <TrezorLink
+                                            typographyStyle="label"
+                                            variant="nostyle"
+                                            href={getNftExplorerUrl(network, nft, id.toString())}
+                                            target="_blank"
+                                            onClick={e => e.stopPropagation()}
+                                        >
+                                            <Icon name="arrowUpRight" size={12} />
+                                        </TrezorLink>
+                                    </Row>
+                                </HiddenPlaceholder>
+                            </Text>
+                        </Table.Cell>
+                    </Table.Row>
+                ))}
+            {nft.type === 'ERC1155' &&
+                nft.multiTokenValues?.map((value, index) => (
+                    <Table.Row
+                        key={nft.contract + index}
```

Comment:
If `nft.contract` is not unique I prefer to use string interpolation

```suggestion
                        key={`${nft.contract}-${index}`}
```

If `nft.contract` is unique, I prefer not to use `index`

Reply:
In this case it's no unique